packages/types/src/documents/global/chat.ts

```diff
@@ -0,0 +1,19 @@
+import { Document } from "../../"
+import type { UIMessage } from "ai"
+
+export interface ChatApp extends Document {
+  title?: string
+  greeting?: string
+  description?: string
+  theme?: string
+  agentId: string
+  live?: boolean
+  settings?: Record<string, any>
+}
+
+export interface ChatConversation extends Document {
+  chatAppId: string
+  userId?: string
```

Comment:
Is user id not required?

Reply:
its because its not there on the first request, ill split the request types out better